chef/cookbooks/crowbar/recipes/prepare-upgrade-scripts.rb

```diff
@@ -81,6 +81,12 @@
 remote_node = roles.include? "pacemaker-remote"
 is_cluster_founder = use_ha && node["pacemaker"]["founder"] == node[:fqdn]
 
+clone_stateless = if node["pacemaker"]
+  node["pacemaker"]["clone_stateless_services_orig"] || true
+else
+  true
+end
```

Comment:
Is true really the correct default to assume here? I am actually not sure which one is "more correct".

Reply:
I think the default for SOC7 is still pacemaker